openedx/adg/common/mailchimp_pipeline/apps.py

```diff
@@ -0,0 +1,11 @@
+"""
+MailchimpPipeline app configurations
```

Comment:
Space missing between MailChimp and Pipeline

Reply:
If this is just a single line we can also write it as
`""" Mailchimp pipeline app configurations """`

Reply:
@ahmed-faraz46 This format is against Edx recommendation